dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/RequestUniqueIdentifier.kt

```diff
@@ -16,11 +16,25 @@ internal fun identifyRequest(request: Request): String {
     val method = request.method()
     val url = request.url()
     val body = request.body()
-    return if (body == null || body == Util.EMPTY_REQUEST) {
+    return if (body == null) {
         "$method•$url"
     } else {
-        val contentLength = body.contentLength()
+        val contentLength = try {
+            body.contentLength()
+        } catch (@Suppress("SwallowedException") ioe: IOException) {
+            0
+        }
         val contentType = body.contentType()
-        "$method•$url•$contentLength•$contentType"
+        // TODO RUMM-3062 It is possible that if requests are say GZIPed (as an example, or maybe
```

Comment:
I thin you are missing something after `as an example` ?

Reply:
GZIP is an example :) I put the bracket at the wrong place, will update it.